apps/nextjs-website/src/app/profile/agreements/page.tsx

```diff
@@ -40,73 +88,127 @@ const Agreements = () => {
+          }}
+        >
+          <Typography
+            variant='body2'
+            sx={{
+              fontSize: '14px',
+              color: palette.text.secondary,
+            }}
+          >
+            {t('newsletter.description')}
+          </Typography>
+          <Box
+            sx={{
+              margin: 0,
+              padding: 0,
+              minWidth: '110px',
+              textAlign: 'right',
+            }}
+          >
+            {hasAcceptedMailingListSubscription ? (
+              <ButtonNaked
+                disabled={isSubscriptionButtonDisabled}
+                sx={{
+                  color: palette.error.dark,
+                  whiteSpace: 'nowrap',
+                }}
+                onClick={handleUnsubscribe}
+              >
+                {t('newsletter.unsubscribe')}
+              </ButtonNaked>
+            ) : (
+              <ButtonNaked
+                disabled={isSubscriptionButtonDisabled}
+                sx={{ whiteSpace: 'nowrap' }}
+                onClick={handleSubscribe}
+              >
+                {t('newsletter.subscribe')}
+              </ButtonNaked>
+            )}
+          </Box>
+        </Box>
+        <Divider sx={{ marginY: '32px' }} />
+        <Typography
+          variant='h6'
+          sx={{
+            marginBottom: '24px',
+            fontSize: '16px !important',
+            fontWeight: '600',
+          }}
+        >
+          {t('privacy.title')}
+        </Typography>
         <Typography
           variant='body2'
           sx={{
+            marginBottom: '24px',
             fontSize: '14px',
             color: palette.text.secondary,
           }}
         >
-          {t('newsletter.description')}
+          {t('privacy.basicData')}
         </Typography>
-        <Box sx={{ margin: 0, padding: 0 }}>
-          <ButtonNaked disabled={true}>{t('newsletter.subscribe')}</ButtonNaked>
-        </Box>
-      </Box>
-      <Divider sx={{ marginY: '32px' }} />
-      <Typography
-        variant='h6'
-        sx={{
-          marginBottom: '24px',
-          fontSize: '16px !important',
-          fontWeight: '600',
-        }}
-      >
-        {t('privacy.title')}
-      </Typography>
-      <Typography
-        variant='body2'
-        sx={{
-          marginBottom: '24px',
-          fontSize: '14px',
-          color: palette.text.secondary,
-        }}
-      >
-        {t('privacy.basicData')}
-      </Typography>
-      <Typography
-        variant='body2'
-        sx={{
-          marginBottom: '24px',
-          fontSize: '14px',
-          color: palette.text.secondary,
-        }}
+        <Typography
+          variant='body2'
+          sx={{
+            marginBottom: '24px',
+            fontSize: '14px',
+            color: palette.text.secondary,
+          }}
+        >
+          {privacyStatement}
+        </Typography>
+      </Stack>
+      <Snackbar
+        open={!!info}
+        autoHideDuration={4000}
```

Comment:
Maybe is better to move this magic number to `apps/nextjs-website/src/config.ts`

Reply:
This is the task: https://pagopa.atlassian.net/browse/DEV-1047